spec/classroom_spec.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../classroom'
+
+RSpec.describe Classroom do
+  let(:classroom_label) { 'Math Class' }
+  let(:student) { Student.new('John Doe') }
+
+  subject(:classroom) { described_class.new(classroom_label) }
+  let(:custom_students) { [] }
+
+  before do
+    allow(classroom).to receive(:students).and_return(custom_students)
+  end
+
+  describe '#initialize' do
+    it 'sets the label correctly' do
+      expect(classroom.label).to eq(classroom_label)
+    end
+
+    it 'initializes an empty students array' do
+      expect(custom_students).to be_empty
+    end
+  end
+
+  describe '#add_student' do
+    it "adds a student to the classroom's students array" do
+      classroom.add_student(student)
+      expect(custom_students).to include(student)
+    end
+
+    it 'assigns the classroom to the student' do
+      classroom.add_student(student)
+      expect(student.classroom).to eq(classroom)
+    end
+  end
+end
```

Comment:
- [ ] For the project requirements, kindly make sure all your tests are passing as right now, some of them in the `Classroom` class are not passing. Please fix thtat to match the requirements. Below is the picture of the result
![Screenshot from 2023-05-26 04-18-13](https://github.com/ismailmunyentwari9/OOP-school-library/assets/83042742/f69d58cd-3383-4c7b-bc77-0eaf08b5217c)

Reply:
@tresorsawasawa Thank you for the response, though on our end when we run the test this is how it shows us, 

![Screenshot from 2023-05-26 13-34-50](https://github.com/ismailmunyentwari9/OOP-school-library/assets/55842414/bea23601-0640-4ae0-b5d9-9634bb1ce8b2)

We are not sure what could be the issue then, kindly let  us know after reviewing it again. Than you aheap.